skopt/searchcv.py

```diff
@@ -268,43 +268,17 @@ class BayesSearchCV(BaseSearchCV):
 
     """
 
-    def __init__(self, estimator, search_spaces=None, optimizer_kwargs=None,
+    def __init__(self, estimator, search_spaces=None, optimizer_kwrgs=None,
```

Comment:
`kwargs` is the standard in python so I think we should keep it

Reply:
Thank you for reviewing.

Agreed, it should be optimizer_kwargs, and all the current optimizer_kwargs should be optimizers_kwargs_.

I'll make a new, clean pass for variable names and refactoring. I think it's going to pass all checks this time.